pom.xml

```diff
@@ -1893,6 +1903,24 @@
         </pluginManagement>
     </build>
 
+    <!-- Enable this only in branches to experiment with snapshot releases from, for example, Infinispan -->
```

Comment:
This does not look very nice. Should we rather add this here: https://github.com/keycloak/keycloak/blob/main/maven-settings.xml hidden behind some profile?